lib/checks/label/multiple-label.js

```diff
@@ -1,12 +1,21 @@
 const id = axe.utils.escapeSelector(node.getAttribute('id'));
-let labels = Array.from(document.querySelectorAll(`label[for="${id}"]`));
 let parent = node.parentNode;
+let root = document;
+if (!axe.utils.contains(node.ownerDocument, node)) {
```

Comment:
I tried `.contains` which fails in the IE integration test (which is strange because it's supported in IE9). So, I changed to the util, which fails in the unit tests. Any ideas?